dracut/31decrypt-root/decrypt-root.service

```
@@ -0,0 +1,16 @@
+[Unit]
+Description=Generate and execute a systemd-cryptsetup service to decrypt the ROOT partition
+
+DefaultDependencies=no
+# If systemd-networkd.service is used in the initramfs, then we should run after it, since the ROOT partition
+# might use network-bound encryption.
+After=systemd-networkd.service dev-disk-by\x2dpartlabel-ROOT.device
+Wants=dev-disk-by\x2dpartlabel-ROOT.device
+Before=dracut-initqueue.service
+
+[Service]
+Type=oneshot
```

```suggestion
Type=oneshot
RemainAfterExit=yes
```
Not sure if we would hit this but that can prevent the service from being executed more that once